<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WgIntListEditor</class>
 <widget class="QWidget" name="WgIntListEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>262</width>
    <height>50</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border:1px solid rgb(171,189,212);
background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:0.999955, y2:0.983, stop:0 rgba(255, 255, 255, 255), stop:1 rgba(188, 220, 250, 255));</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QLabel" name="labName">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>190</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>190</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(35, 62, 95);
border: none;</string>
     </property>
     <property name="text">
      <string>Attr</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboBoxIndex">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::WheelFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox {
	border:1px solid rgb(168, 186, 209);
	border-radius: 5px;
	background: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
}

QComboBox::hover
{
	background-color: rgb(255, 255, 191);
}

QComboBox QAbstractItemView {
    border:1px solid rgb(238, 107, 71);
    background-color: rgb(255, 255, 191);
    selection-background-color: rgb(238, 107, 71);
}
</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="spinBoxValue">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QSpinBox {
	border:1px solid rgb(168, 186, 209);
	border-radius: 5px;
	background: rgb(255, 255, 255);
	color: rgb(0, 0, 0);
}

QSpinBox::hover
{
	background-color: rgb(255, 255, 191);
}

QSpinBox QAbstractItemView {
    border:1px solid rgb(238, 107, 71);
    background-color: rgb(255, 255, 191);
    selection-background-color: rgb(238, 107, 71);
}</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
